src/tsickle.ts

```diff
@@ -73,6 +73,11 @@ export interface TsickleHost extends googmodule.GoogModuleProcessorHost,
    */
   googmodule: boolean;
 
+  /**
```

Comment:
I don't love this name, it's more like "don't change the module".
Maybe we should just do this when `transformTypesToClosure` is true but not `googmodule`?

Reply:
Updated to reflect this